conduct.md

```diff
@@ -212,4 +214,7 @@ Any decision made by the instructor for the given infraction(s) will be final.
 ## Names of Individuals who Agree to Adhere to the Guide
 
 Janyl Jumadinova
-:heavy_exclamation_mark: TODO: Add your name
+Lex Caldwell
```

Comment:
Please do not remove this `TODO` until members from other groups have already added their names

Reply:
I told them to do that so that is my fault!

Reply:
No worries